src/sql/src/query_model/hir.rs

```diff
@@ -379,6 +380,26 @@ impl FromHir {
                     position: 0,
                 })
             }
+            HirScalarExpr::Windowing(expr) => {
+                let func = match expr.func {
+                    crate::plan::expr::WindowExprType::Scalar(scalar) => {
+                        crate::query_model::scalar_expr::WindowExprType::Scalar(scalar.func)
```

Comment:
Is it ok to lose the `scalar.order_by` information? If it is, why does `plan::expr::ScalarWindowExpr` have the `order_by` information in the first place?

Reply:
Yes, it is Ok to lose it. That information is redundant. In Hir, for some reason, we have `order_by` info is divided into `order by expr` and `order by column positions` which are the column positions those expression will be mapped. In QGM, we just need the expression. CSE is something that belongs in a later stage.